runelite-client/src/main/java/net/runelite/client/events/ChatboxInput.java

```diff
@@ -25,12 +25,14 @@
 package net.runelite.client.events;
 
 import lombok.Data;
+import lombok.AllArgsConstructor;
 
 @Data
+@AllArgsConstructor
```

Comment:
Looks like you are never creating the object? So i guess this change is unnecessary too. Also I think data already has AllArgsConstructor

Reply:
Either a constructor is required or @AlllArgsConstructor is needed.

Data doesn't generate it unless the variables are final.